server/routes/index.ts

```diff
@@ -775,8 +775,8 @@ export function defineRoutes(router: IRouter) {
           index: request.body.index.join(','),
           ignore_unavailable: true,
           allow_no_indices: true,
-          body: request.body,
```

Comment:
Is this intentional?

Reply:
yes, the backend API doesn't require body